### ISSUE TITLE
Clicking on node

### ISSUE THREAD

**Issue by VictorBjelkholm** [jbenet/depviz#30]
Currently, this is the behavior of clicking on a node:

**Github Icon**: Go to github.com
**Issue title**: Go to that issue on github.com
**Clicking/Hovering depends/related/blocks icons**: Nothing

Suggested behavior:

**Github Icon**: Open new tab with issue on github.com
**Issue title**: Navigate to that issue in depviz
**Clicking/Hovering depends/related/blocks icons**: Show what the icon means

**Comment by jbenet** [comment]
I agree with these o/ -- said more in https://github.com/jbenet/depviz/issues/45

**Comment by wking** [comment]
On Tue, Nov 29, 2016 at 03:13:05AM -0800, ᴠɪᴄᴛᴏʀ ʙᴊᴇʟᴋʜᴏʟᴍ wrote:
> **Github Icon**: Open new tab with issue on github.com
> **Issue title**: Navigate to that issue in depviz

I like having the host icon link to the host, because that's what the
icon represents.  I expect the default meaning of links to be “link to
${link-content}”, not “link to this issue on ${link-content}”.
Similarly, we have the person image linking to that person's page, and
not to the issue somehow in the context of that person's page/site.
With this approach, the ID (e.g. jbenet/depviz#45) is the obvious
choice for linking to the native issue itself (e.g. [1]).

The problem with my approach is that all the real estate on collapsed
cards is gone, and we still need to bind “center on this card and
update the location” to something.  #45 proposed a new
permalink/anchor button to get some space for this, but it doesn't
seem to be gaining traction.

@VictorBjelkholm posted an example of how Waffle handles links to the
native issue (with the host icon and the comment icon both linking to
the native issue) [2], but that doesn't seem like an intuitive choice
to me (why the host icon, but not the person icon?  Why the comment
icon, but not the title?).  But consistency and intuition aside, I
agree that a link to the host is less interesting than a link to the
associated people.  So I'm not very strongly against the proposal
here.  I just feel like there should be an approach that gives us a
clear “center…” action without subverting the host logo.

I think the “link to this issue on ${link-content}” intepretation is
more obvious if there are *choices*.  That's a bit hard for us, since
a GitHub issue is probably only going to exist on GitHub (for example)
and not also be mirrored on some other host.  Maybe you could link to
GHTorrent or whatever?  And if we have a small, boxy depviz logo (#14)
in that list, maybe it would be a good anchor for “center…”.  So the
collapsed card would hold something like:

  ${ID} ${host-logo} ${depviz-logo} [${mirror-logo}…] 

e.g.

  jbenet/depviz#45 🐙ⓥ🐈

Then you could have the whole “${ID} ${host-logo}” span link to the
native issue, the depviz logo being “center…” (i.e. link to the issue
on depviz), and the mirror logos being “link to this issue on the
${mirror}”.

Or maybe I'm just overthinking this, and folks will get used to
whatever we pick ;).

[1]: https://github.com/jbenet/depviz/issues/45
[2]: https://github.com/jbenet/depviz/issues/45#issuecomment-265147473
